src/core/src/FabricPrepareRenderResources.cpp

```diff
@@ -55,9 +55,8 @@ struct TileLoadThreadResult {
     glm::dmat4 tileTransform;
 };
 
-bool hasBaseColorTextureGltf(const FabricMesh& fabricMesh) {
-    return fabricMesh.material != nullptr && fabricMesh.material->getMaterialDefinition().hasBaseColorTexture() &&
-           fabricMesh.materialInfo.baseColorTexture.has_value();
+bool hasBaseColorTexture(const FabricMesh& fabricMesh) {
+    return fabricMesh.material != nullptr && fabricMesh.material->getMaterialDefinition().hasBaseColorTexture();
 }
```

Comment:
The deleted part used to be important, but is no longer needed after https://github.com/CesiumGS/cesium-omniverse/pull/495.